webapp/app/js/perf.js

```diff
@@ -149,6 +149,11 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
       $scope.$digest();
     };
 
+    Mousetrap.bind('escape', function() {
+      $scope.ttLocked = false;
+      $scope.hideTooltip();
+      });
```

Comment:
I think maybe this closing brace can be indented 2 chars to the left? So it lines up under the M.